File: patch_ui.py
import os

FRONTEND_DIR = '/Users/eugeniucazmal/Downloads/dev_office/pontaj_digital/frontend/src'

emojis = ['📋', '👋', '✏️', '📝', '🔔', '👥', '📦', '🕐', '☕', '🔴', '🔄', '✅', '💡', '📍', '⚠️', '📊', '🚚', '✍️', '⚙️', '📤', '📎', '📷']

def process_file(filepath):
    try:
        with open(filepath, 'r', encoding='utf-8') as f:
            content = f.read()

        initial_content = content

        for emoji in emojis:
            content = content.replace(emoji + ' ', '')
            content = content.replace(emoji, '')

        content = content.replace('rounded-lg', 'rounded-full')
        content = content.replace('rounded-md', 'rounded-full')

        if content != initial_content:
            with open(filepath, 'w', encoding='utf-8') as f:
                f.write(content)
            print(f"Updated: {filepath}")
    except Exception as e:
        print(f"Error reading {filepath}: {e}")

def main():
    count = 0
    for root, dirs, files in os.walk(FRONTEND_DIR):
        for file in files:
            if file.endswith('.jsx') or file.endswith('.js'):
                count += 1
                process_file(os.path.join(root, file))
    print(f"Total files checked: {count}")

if __name__ == '__main__':
    main()

File: patch_ui.js
const fs = require('fs');
const path = require('path');

const FRONTEND_DIR = '/Users/eugeniucazmal/Downloads/dev_office/pontaj_digital/frontend/src';

const emojisToRemove = [
    '📋', '👋', '✏️', '📝', '🔔', '👥', '📦', '🕐', '☕', '🔴', '🔄', '✅', '💡', '📍', '⚠️', '📊', '🚚', '✍️', '⚙️', '📤', '📎', '📷'
];

function processFile(filePath) {
    const ext = path.extname(filePath);
    if (ext !== '.jsx' && ext !== '.js') return;

    let content = fs.readFileSync(filePath, 'utf8');
    let initialContent = content;

    // 1. Remove emojis
    for (const emoji of emojisToRemove) {
        // Remove emoji and following space if it exists
        content = content.split(emoji + ' ').join('');
        // Also remove if no following space
        content = content.split(emoji).join('');
    }

    // 2. Replace rounded-lg and rounded-md with rounded-full
    // We will do a generic replacement of rounded-lg and rounded-md to rounded-full
    // Wait, some cards might be rounded-lg. Is it safe to just replace all?
    // The user said: "nici macar nu este rotunjit cum am specificat eu in regului... fiecare bara, fiecrae buton. sa corespund cau regul ui ui."
    // And earlier "debulel si scoate emoji". 
    // Usually, input and button classes contain "px-", "py-", "w-full", etc. 
    // Let's replace specifically in classNames that look like buttons or inputs.
    // Instead of regex parsing tags, let's just do a naive replace and rely on the premium UI rules.
    // But wait, cards should probably be rounded-xl or rounded-2xl. rounded-lg is mostly used for inputs/buttons in this codebase.
    // Let's replace 'rounded-lg' with 'rounded-full' globally for any JSX file in `src/pages` and `src/components`.
    
    content = content.replace(/rounded-lg/g, 'rounded-full');
    content = content.replace(/rounded-md/g, 'rounded-full');

    if (content !== initialContent) {
        fs.writeFileSync(filePath, content, 'utf8');
        console.log('Updated:', filePath);
    }
}

function walkDir(dir) {
    const files = fs.readdirSync(dir);
    for (const file of files) {
        const fullPath = path.join(dir, file);
        if (fs.statSync(fullPath).isDirectory()) {
            walkDir(fullPath);
        } else {
            processFile(fullPath);
        }
    }
}

walkDir(FRONTEND_DIR);
console.log('Done patching UI.');
